components/profile-controller/controllers/profile_controller.go

```diff
@@ -588,3 +591,17 @@ func removeString(slice []string, s string) (result []string) {
 	}
 	return
 }
+
+func updateNamespaceLabels(ns *corev1.Namespace) bool {
```

Comment:
nit could we add a unittest for this.

Reply:
I notice there is no any ut in this component. I think maybe we can add another PR to handle UT for this component.

Reply:
Would it be that much work to add one in this PR? Your adding a new function updateNamespaceLabels and I wouldn't expect a unittest function to have much todo with unittests for any other functions.